### PR TITLE
MC-4995: add Partnership (JobDetails)

### PR DESCRIPTION
**Pull request by drewwills**

### CODE REVIEW

**Review by SandiMixam — APPROVED** [review]

File: src/main/java/com/mixam/mxjdf4/sdk/OrderType.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.mixam.mxjdf4.sdk;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum OrderType {
+
+    STANDARD(0),
+    FOURTHWALL(1),
```
Comment (ashley-mixam) [inline src/main/java/com/mixam/mxjdf4/sdk/OrderType.java:8]: I think we should POD in here as well

File: README.md [hunk 2]
```diff
@@ -288,20 +288,22 @@ will match the trim box of the actual artwork for the just jacket.
...
-|assumedPrintType|The expect print technology|NONE(0), <br>LITHO(1), <br>DIGITAL(2), <br>INKJET(3)|
-|jobType|Classification of print job|PRIMARY_PRINT(0), <br>RIPPED_PROOF(1), <br>HARD_COPY_PROOF(2)|
-|remark|Remarks for the supplier added by Mixam at fulfilment-time, if any (String)||
-|machineName|The name of the recommended machine for this job identified by our pricing engine (String)||
+|additionalProjectName| | Additional Order number. Reference number. (String)                                         ||
+|completionType| Urgency of the job. Default value is 4 (Standard) | INVALID(0), <br>SAME_DAY(1), <br>EXPRESS(2), <br>OVERNIGHT(3), <br>STANDARD(4), <br>SAVER(5); |
+|totalCirculation| The total number of copies (Integer) | |
+|assumedPrintType| The expected print technology | NONE(0), <br>LITHO(1), <br>DIGITAL(2), <br>INKJET(3) |
+|jobType| Classification of print job | PRIMARY_PRINT(0), <br>RIPPED_PROOF(1), <br>HARD_COPY_PROOF(2) |
+|orderType| Classification of order, if applicable | STANDARD(0), <br>FOURTHWALL(1) |
```
Comment (ashley-mixam) [inline README.md:304]: Thats a better name for this

**Review by ashley-mixam — APPROVED** [review]

File: src/main/java/com/mixam/mxjdf4/sdk/OrderType.java (hunk 1 above)
Comment (drewwills) [reply src/main/java/com/mixam/mxjdf4/sdk/OrderType.java]: ✅

File: README.md (hunk 2 above)
Comment (drewwills) [reply README.md]: It's too general... but it's better in other ways.

**Pull request merged**